scripts/test_tf2_execution.py

```diff
@@ -221,13 +223,24 @@ def test_checkpointed(train_ds, test_ds, solver, epochs=1):
     sns.set_style("darkgrid")
 
     train_ds, test_ds = get_data(dataset, batch_size=batch_size)
-    data = {
-        "baseline": (test_baseline(train_ds, test_ds, n_epochs)),
-        "checkpoint_all": (test_checkpointed(train_ds, test_ds, solve_checkpoint_all, epochs=n_epochs)),
-        "checkpoint_sqrtn_ap": (test_checkpointed(train_ds, test_ds, solve_chen_sqrtn_ap, epochs=n_epochs)),
-        # "checkpoint_sqrtn_noap": (test_checkpointed(train_ds, test_ds, solve_chen_sqrtn_noap, epochs=n_epochs)),
-    }
+    memdmp = "TF_BFC_MEMORY_DUMP"
```

Comment:
This requires recompilation, so there should be a check to determine if the installed TensorFlow version has the memory logging flags. If so, it's good to run the test. But if not, it should be skipped.

Reply:
This code will not error out if there is no recompilation, it will just do nothing.  Will remove from the actual software artifact though, my bad!.